### ISSUE TITLE
Chaning some other variables?

### ISSUE THREAD

**Issue by emdeex** [robbiewain/theopenbudget#4]
Hi,

Thanks for your help so far.

I'm making progress in adapting this for other types of government departments.  Can I modify the UI thats says "Cost per Australian person".. to something more meaningful in my case, such as "Cost per Ratepayer"?  ...and adjust the metric accordingly?

Also modify the default search box suggestion... "eg. Health", and change the "theopenbudget.org" heading?

If these were made variables in a config file, that would be cool.

regards

**Comment by robbiewain** [comment]
@emdeex I agree moving those variables out into a config file would make the visualisation more reusable.

We have no current plans to do this though but pull requests are very welcome.
